### DISCUSSION TITLE
Namespace Error of Reload

### DISCUSSION THREAD

**Discussion by douglas717** [remix-run/remix#7481] [Q&A]
Looking for some advice on how to fix

Windows 10 

Run npm run dev 
site builds and Remix App Server started at http://localhost:3000 .,,, no problem 

When I make a change to a page the LiveServer rebuilds the site and the following error occurs 

node_modules\pidtree\lib\bin.js:54   
      return done(new Error(stderr));
                  ^
Error: ERROR:
Description = Invalid namespace
    at ChildProcess.<anonymous>

Only way to get out is to terminate the process and run npm run dev again

**Comment by thatdevclaude** [comment]
have you found the solution yet?

**Reply by douglas717** [reply]
Unfortunately no
